server/app.js

```diff
@@ -0,0 +1,36 @@
+const express = require('express');
+const app = express();
+const db = require('./db.js');
+
+app.use(express.static('./public/dist'));
+
+app.get('/api/users', (req, res) => {
+  db.User.findAll()
+    .then((users) => {
+      res.send(users);
+    });
```

Comment:
And if there are no users? What does sequalize return? null, 0, undefined, error?

Reply:
It should be returning null